### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by bhageena**

### CODE REVIEW

File: postman/README.md [hunk 1]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
 nim update
 ```
 
+## Manage Postman API Keys
+
+Key management command facilitates to have multiple postman API secrets to be stored as key value pair, and allows one of them to be used as current.
```
Comment (rabbah) [inline postman/README.md]:
```suggestion
Key management with `nim auth postman` facilitates working with one or more Postman API keys.
```

File: postman/README.md [hunk 2]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
...
+It also allows to switch between the keys using `switch` command and thus becoming the current key.
```
Comment (rabbah) [inline postman/README.md]:
```suggestion
You can switch conveniently between them using the `nim auth postman switch` command.
```

File: postman/README.md [hunk 3]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
...
+The current key is then picked up automatically without user needing to type it in manually.
+
+Adding the first key makes it current:
```
Comment (rabbah) [inline postman/README.md]:
```suggestion
Adding a key automatically makes it current.
```

File: postman/README.md [hunk 4]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
...
+
```
Comment (rabbah) [inline postman/README.md:258]:
```suggestion
Here is an example of how to add a Postman API key:
```

File: postman/README.md [hunk 5]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
...
+```
+nim auth postman -a --name mykey --key PMAK-5e2a993188ce8e003888f36b-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+the postman key with name 'mykey' was added and is now current
```
Comment (rabbah) [inline postman/README.md]: Can you capitalize the printed message so postman is Postman instead?

```suggestion
the Postman key with name 'mykey' was added and is now current.
```

File: postman/README.md (hunk 3 above)
Comment (rabbah) [reply postman/README.md]: is it only the first key that becomes current or any added key?

File: postman/README.md [hunk 6]
```diff
@@ -242,6 +246,37 @@ Plugins will auto update alongside the CLI, but to trigger an update directly ru
...
+```
+
+Following are the available options.
```
Comment (rabbah) [inline postman/README.md]:
```suggestion
Here are the available options.
```

File: postman/README.md (hunk 3 above)
Comment (bhageena) [reply postman/README.md]: it's any added key.
thanks!

**Pull request merged**